Mathlib/Analysis/Complex/Conformal.lean

```diff
@@ -58,10 +58,8 @@ theorem isConformalMap_complex_linear {map : ℂ →L[ℂ] E} (nonzero : map ≠
     simp only [map.coe_coe, map.map_smul, norm_smul, norm_inv, norm_norm]
     field_simp only [one_mul]
   · ext1
-    -- porting note (#10745): was simp
-    rw [coe_restrictScalars', coe_smul', LinearIsometry.coe_toContinuousLinearMap,
-      LinearIsometry.coe_mk, Pi.smul_apply, LinearMap.smul_apply, LinearMap.coe_restrictScalars,
-      coe_coe, smul_inv_smul₀ minor₁]
+    -- porting note (#10745): was `simp`; added simp lemma
```

Comment:
Also clarified this note: this proof fails without specifying `smul_inv_smul₀ minor₁` (which is tagged simp).

Reply:
Supplied here also